### ISSUE TITLE
NegPosRatio for sampling

### ISSUE THREAD

**Issue by harshakokel** [hayesall/srlearn#68]
**Describe if the Java version of BoostSRL already supports this.**
BoostSRL java version has a parameter -NegPosRatio, it would be great to have support of that parameter in the srlearn.

**Describe the solution you'd like**
Perhaps an extra parameter in the BoostedRDN constructor for mentioning NegPosRatio?

**Comment by hayesall** [comment]
100% agree on `negPosRatio`.

Is there a use case / workflow you'd like for `combine`? For example: I tend to find it more useful for interpreting results than using for inference.

**Comment by harshakokel** [comment]
Yes, the combined tree is mostly for interpreting results. Looking at the combined clauses.

**Comment by hayesall** [comment]
`negPosRatio` in #69, still need to think on the combined case.

**Comment by harshakokel** [comment]
Also in case you do support combined tree, I suggest we rename it to something else. "Combined" term is misguiding.
